Plugin.pm

```diff
@@ -1378,6 +1388,34 @@ sub QobuzGetTracks {
 				nextWindow => 'parent'
 			};
 
+			if (my $item = trackInfoMenuBooklet($client, undef, undef, $album)) {
+				push @$items, $item;
+			}
+
+			# Build a consolidated list of all artists on the album
+			my @uniquePerformers;
+			if ( scalar @performers ) {
+				my %seen = ();
+				my $tracks;
+				foreach my $item (@performers) {
+					push @{$tracks->{$item->{'name'}}->{'tracks'}}, $item->{'track'};
+					delete $item->{'track'};
+					push(@uniquePerformers, $item) unless $seen{$item->{'name'}}++;
+				}
+				foreach my $item (@uniquePerformers) {
+					my @tracks = @{$tracks->{$item->{'name'}}->{'tracks'}};
+					if ( @tracks && scalar @tracks < $album->{tracks_count} ) {
+						$item->{'name'} .= scalar @tracks == 1 ? " (track " : " (tracks ";
```

Comment:
Shouldn't "track"/"tracks" be localized?

Reply:
Yes.